### ISSUE TITLE
Does evaluate() always return 0 result?  

### ISSUE THREAD

**Issue by nanzhao** [bojone/sohu2021-baseline#2]
During the training, It seems evaluate() always returns 0 result.

so if it has problems with these accuracy caculation:
        total_a += ((y_pred + y_true) * (flag == 0)).sum()
        right_a += ((y_pred * y_true) * (flag == 0)).sum()
        total_b += ((y_pred + y_true) * (flag == 1)).sum()
        right_b += ((y_pred * y_true) * (flag == 1)).sum()
should be?
        total_a += (flag == 0).sum()
        right_a += ((y_pred == y_true) * (flag == 0)).sum()
        total_b += (flag == 1).sum()
        right_b += ((y_pred == y_true) * (flag == 1)).sum()

and f1 should remove "2*" also :
    f1_a = right_a / total_a
    f1_b = right_b / total_b

**Comment by hurun** [comment]
Evaluator函数注释掉        
    # optimizer.apply_ema_weights()
   # optimizer.reset_old_weights()
能改变0的问题

**Comment by bojone** [comment]
代码没错，我算的是f1不是acc。另外对齐环境，

**Comment by nanzhao** [comment]
我的环境，keras==2.3.1, tensorflow-GPU==2.2.0，训练没问题的，能够正常收敛，显示acc 0.7多，但就是每次评估的时候总是0，所以weight保存不下来。
